docs/setting_up_aml.md

```diff
@@ -64,8 +64,10 @@ You can skip this if you have chosen automatic deployment above.
 
 We recommend using [low priority](https://docs.microsoft.com/en-us/azure/batch/batch-low-pri-vms) clusters, since 
 they only cost a fraction of the dedicated VMs.
-As a reference, the Prostate model and the Head and Neck model require VMs with 4 GPUs with at least 16GB of memory
-per GPU, for example `Standard_ND24s`, `Standard_NC24s_v3` or `Standard_NC24s_v2`.
+As a reference:
+* The Prostate model and the Head and Neck model require VMs with 4 GPUs with at least 16GB of memory
```

Comment:
VMs with 4GPUs with at least 24GB of memory (ND24s) are required for all models, in fact.
Maybe here also say what to do if not fitting? Reduce train_batch_size, and if that's not enough, reduce train_crop_size and test_crop_size. Adjust inference_stride_size accordingly

Reply:
Makes, sense, I will need to check, as lowering the train_batch_size (8 --> 4) did not fix it for me

Reply:
Maybe ND24s should be mentioned as an example, to get the reader started quickly without having to analyze compute cluster SKUs.